deployment-templates/compute-engine/explain-instance.py

```diff
@@ -46,6 +48,18 @@ def GenerateConfig(context):
 
     resources = []
 
+    EXPORT_INITIALIZE_VARS = (
+        'export SQL_PORT={0}\n'
+        'export SQL_INSTANCE_CONN_STRING="{1}"\n'
+        'export EXPLAIN_DB_NAME="{2}"\n'
```

Comment:
I recommend replacing the use of EXPLAIN_DB with FORSETI_DB.

Reply:
I fixed this one. Can we open issues if there are other names needs to be changed? I don't want to accumulate too much fixes and improvements in this PR.

Reply:
I'm fine with putting the variable rename changes in a future PR.